build/init.yml

```diff
@@ -27,4 +27,4 @@ steps:
     command: custom
     verbose: false
     customCommand: 'install -g vsce'
- 
\ No newline at end of file
+ 
```

Comment:
I think the blank lines were added when @anpaz was trying to re-trigger the CI build.

```suggestion
```